smoke-tests/src/test/java/com/splunk/opentelemetry/AppServerTest.java

```diff
@@ -66,15 +67,20 @@ protected void assertWebAppTrace(ExpectedServerAttributes serverAttributes)
         1, traces.countFilteredAttributes("http.url", url), "The span for the initial web request");
     Assertions.assertEquals(
         2,
-        traces.countFilteredAttributes("http.url", "http://localhost:8080/app/headers"),
+        traces.countFilteredAttributes("http.url", getUrl("/app/headers", true)),
         "Client and server spans for the remote call");
 
+    if (testImage.isProprietaryImage) {
```

Comment:
Do we need this `WithSpan`? Or more precisely: why do we need test application which differs from Otel's one? We can publish Otel testing app and reuse it. Will make our tests simpler.

As first step we can remove `WithSpan` method from our testing app.

Reply:
This was present before, and I'd rather leave any functional changes to the test application out of the scope of this PR, especially since it may warrant further discussion that is not relevant to the topic of this PR.